lib/fluent/plugin/in_object_space.rb

```diff
@@ -31,50 +33,20 @@ def initialize
     config_param :tag, :string
     config_param :top, :integer, default: 15
 
-    class TimerWatcher < Coolio::TimerWatcher
-      def initialize(interval, repeat, log, &callback)
-        @callback = callback
-        @log = log
-        super(interval, repeat)
-      end
-
-      def on_timer
-        @callback.call
-      rescue
-        # TODO log?
-        @log.error $!.to_s
-        @log.error_backtrace
-      end
-    end
-
     def configure(conf)
```

Comment:
Could you remove this method definition? No need to leave it here (just calling `super`).